includes/options/class-amp-options-menu.php

```diff
@@ -205,15 +211,51 @@ public function render_theme_support() {
 				<dd>
 					<p><?php echo wp_kses_post( $reader_description ); ?></p>
 
-					<p>
-						<?php
-						/* translators: %s is the reader theme */
-						echo esc_html( sprintf( __( 'The "%s" theme is currently selected in the wizard. The ability to change it from the admin screen will come soon.', 'amp' ), AMP_Options_Manager::get_option( Option::READER_THEME ) ) );
-						?>
-					</p>
+					<div class="notice notice-info notice-alt inline">
+						<p>
+							<?php
+							/* translators: %s is the reader theme */
+							echo esc_html( sprintf( __( 'The "%s" theme is currently selected in the wizard. The ability to change it from the admin screen will come soon.', 'amp' ), AMP_Options_Manager::get_option( Option::READER_THEME ) ) );
```

Comment:
I'm wondering if we should show the theme name here instead of its slug :thinking:. The user would more than likely not familiar with the installed theme slugs, and in some cases the theme slug might not easily identify the theme in question.

Reply:
Agreed. This is just temporary since all of this will be eliminated with the work that @johnwatkins0 is doing.